app/view_models/auction_show_view_model.rb

```diff
@@ -254,11 +234,7 @@ def for_small_business?
   end
 
   def current_user_can_bid?
-    if auction.type == 'single_bid' && user_bids.any?
-      false
-    elsif over?
-      false
-    elsif future?
+    if !available? || auction.type == 'single_bid' && user_is_bidder?
```

Comment:
this might not be within scope of this PR, but this kind of logic I think should be completely outsourced to a `Rules` class.